prowler/lib/mutelist/mutelist.py

```diff
@@ -449,3 +437,23 @@ def is_item_matched(matched_items, finding_items, tag=False) -> bool:
                 f"{error.__class__.__name__} -- {error}[{error.__traceback__.tb_lineno}]"
             )
             return False
+
+    @staticmethod
+    def validate_mutelist(mutelist: dict) -> tuple[bool, dict]:
+        """
+        Validate the mutelist against the schema.
+
+        Args:
+            mutelist (dict): The mutelist to be validated.
+
+        Returns:
+            tuple[bool, dict]: A tuple containing a boolean indicating if the mutelist is valid and the mutelist itself.
+        """
+        try:
+            validate(mutelist, schema=mutelist_schema)
+            return True, mutelist
```

Comment:
Why we need to return a boolean and a dict and not only a dict? It is a requirement?

Reply:
![Screenshot 2025-06-10 at 17 07 39](https://github.com/user-attachments/assets/8b7ff890-1650-4971-be30-9fd98f7f0867)
this was from this assert, I didn't want to change the logic of the validation at all.